app/models/user.rb

```diff
@@ -4,7 +4,7 @@ class User < ApplicationRecord
   validates :email, uniqueness: true
   validates :role, presence: true
   validate :password_regex
```

Comment:
Not seen this before, how does this work?

Reply:
What are you asking about here?
The only added line is `has_many :profiles`

Reply:
sorry, think comment goes beneath quoted line by default but still confusing 'validate :password_regex', might not even be your code though!